src/actions/fetch-data-resource.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios'
+
+export async function fetchResource (id) {
+
+  const instance = axios.create({
+    transformRequest: [
+      (data, headers) => {
+        headers.common.Accept = 'application/sparql-results+json'
+        return data
+      },
+    ]
+  })
+
+  const query =
+    `
+    DESCRIBE <http://qa.stad.gent/id${id}> <https://qa.stad.gent/id${id}>
+    `
+
+  let formData = new FormData()
+  formData.set('query', query)
+
+  let response
+
+  try {
+    response = await instance({
+      method: 'post',
+      url: 'https://qa.stad.gent/sparql',
+      data: formData,
+    })
+  }
+  catch (e) {
+    throw e
+  }
+
+  response = response.data.results.bindings
```

Comment:
Did you test this? Isn't this possibly going to throw an error? 🤔

Reply:
If the call was successful, data.result.bindings is always present.
Anyway, errors are catched in the calling function.
@nillusius?